File: programs-basic-java/src/test/java/com/local/numbertypes/TechNumberTest.java
package com.local.numbertypes;

import static com.local.numbertypes.NumberType.TECH_NUMBER;

import java.util.List;

import org.junit.jupiter.api.Test;

class TechNumberTest extends AbstractNumberTest {

	@Test
	void testTechNumbers() {
		testNumbersTypeByFile(TECH_NUMBER, "technumbers.txt");
	}

	@Test
	void testTechNumbersByList() {
		// Pass the order as 5
		testNumbersTypeByList(TECH_NUMBER, List.of(81,2025,3025), 3);
	}
}

File: programs-basic-kotlin/src/test/kotlin/com/local/numbertypes/TechNumberTest.kt
package com.local.numbertypes
import org.junit.jupiter.api.Test
class TechNumberTest : AbstractNumberTest() {
	@Test
	fun testTechNumbers() {
		testNumbersTypeByFile(com.local.numbertypes.NumberType.TECH_NUMBER, "technumbers.txt")
	}

	@Test
	fun testTechNumbersByList() {
		// Pass the order as 5
		testNumbersTypeByList(com.local.numbertypes.NumberType.TECH_NUMBER, java.util.List.of<Int>(81, 2025, 3025), 3)
	}
}